### DISCUSSION TITLE
Can not connect to any Wallet on Android as well as IOS

### DISCUSSION THREAD

**Discussion by PPnaruto** [rainbow-me/rainbowkit#2174] [Q&A] [answered]
I am trying to raise PR for the bestWallet to integrate with RainbowKit. While testing the changes, I have came across, I am not able to connect any of the wallet on Android as well as IOS.
Please have a look on attached video.
Is it related to projectId ?
I have added Project Id in the env as well, still facing this issue

https://github.com/user-attachments/assets/620095a4-f7d1-41b0-ba36-5f5a28f5b693

**Comment by DanielSinclair** [comment, marked as answer]
@PPnaruto WalletConnect requires that you whitelist the domain name, which would explain these failing network requests and behavior. When you raise the PR, we can deploy on a whitelisted domain that will allow you to test.

**Comment by PPnaruto** [comment]
Okay, great!, Thanks for the update @DanielSinclair.
